.github/workflows/packaging/install_dependencies.sh

```diff
@@ -6,18 +6,19 @@ apt update \
     git \
     libssl-dev \
     libusb-1.0-0-dev \
-    pkg-config \
-    libgtk-3-dev \
-    libgl1-mesa-dev \
-    libglu1-mesa-dev
+    pkg-config
 
 ## Build and install Intel RealSense SDK 2.0 v2.45.0 ##
 git clone https://github.com/IntelRealSense/librealsense.git -b v2.45.0
 
 PWD=$(pwd)
 
 cd librealsense
-cmake -Bbuild -DCMAKE_BUILD_TYPE=Release -DBUILD_EXAMPLES=false -H.
+cmake -Bbuild \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DBUILD_EXAMPLES=false \
+  -DBUILD_GRAPHICAL_EXAMPLES=false \
+  -DBUILD_TOOLS=false -H.
```

Comment:
I actually would build the tools. those might be helpful for debugging

Reply:
We don't need them here. This is just for the `ros_cross_compile` to fetch `librealsense` and be able to compile. The minimum here is the key so to speed up the build time.

The options to be set on the `librealsense` build are to be updated in https://github.com/Auterion/third_party_packages/tree/master/librealsense

Reply:
ok